webpack/envs/clientDevelopmentConfig.js

```diff
@@ -55,16 +54,18 @@ export const clientDevelopmentConfig = {
     ...standardPlugins,
     new webpack.HotModuleReplacementPlugin(),
     new ForkTsCheckerWebpackPlugin({
-      checkSyntacticErrors: true,
-      formatter: "codeframe",
-      formatterOptions: "highlightCode",
-      watch: ["./src/v2"],
```

Comment:
Non-blocking - `watch` was pointed at v2 to speed things up a bit, since most product devs never leave the `v2` folder

Reply:
Literally doesn't exist anymore, from what I can gather in their changelog it never really worked, caused major performance issues, and duplicated watch functionality that was already in place.